src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -2,46 +2,193 @@
 
 import java.util.List;
 
+@SuppressWarnings("unchecked")
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final double GROWTH_FACTOR = 1.5;
+    private static final int MAX_LIST_SIZE = 10;
+    private Node<T>[] nodes;
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        nodes = new Node[MAX_LIST_SIZE];
+    }
```

Comment:
```suggestion
```

Reply:
So I need not to use Arrays at all in the solution?